nodes/index.adoc

```diff
@@ -129,3 +129,53 @@ As an administrator, You can perform various tasks on a Linux container, such as
 {product-title} provides specialized containers called xref:../nodes/containers/nodes-containers-init.adoc#nodes-containers-init[Init containers]. Init containers run before application containers and can contain utilities or setup scripts not present in an application image. You can use an Init container to perform tasks before the rest of a pod is deployed.
 
 Apart from performing specific tasks on nodes, pods, and containers, you can work with the overall {product-title} cluster to keep the cluster efficient and the application pods highly available.
+
+[id="commonterms-node"]
+== Common terms
+
+This glossary defines common terms that are used in the _node_ content. These terms help you understand _nodes_ effectively.
+
+Container::
+It is a lightweight and executable image that comprises software and all its dependencies. Containers virtualize the operating system, as a result, you can run containers anywhere, from a data center to a public or private cloud to even a developer's laptop.
+
+Daemon set::
+It ensures that a replica of the pod runs on eligible nodes in an {product-title} cluster.
+
+egress::
+Is outgoing traffic from a pod. It is the process of data sharing externally through a network’s outbound traffic.
+
+garbage collection::
+It is the process of cleaning up cluster resources, such as terminated containers and images that are not referenced by any running pods.
+
+Horizontal Pod Autoscaler(HPA)::
+Implemented as a Kubernetes API resource and a controller. You can use the HPA to specify the minimum and maximum number of pods you want to run. You can also specify the CPU or memory utilization that your pods should target.
+
+Ingress::
+Is incoming traffic to a pod.
+
+Job::
+A process that runs to completion. A job creates one or more pod objects and ensures that the specified pods are successfully completed.
+
+Labels::
+Is a key value pair attached to an object such as a pod. You can use labels to organize and select subsets of objects.
+
+Node::
+A worker machine in the {product-title} cluster. A node can be either be a virtual machine (VM) or a physical machine.
+
+Node Tuning Operator::
+You can use the Node Tuning Operator to manage node-level tuning by using the TuneD daemon. It ensures custom tuning specifications are passed to all containerized TuneD daemons running in the cluster in the format that the daemons understand.
+The daemons run on all nodes in the cluster, one per node.
```

Comment:
looks like this sentence hard-wrapped. It renders fine in the preview, but should be fixed per our docs guidelines. https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#general-file-guidelines